### ISSUE TITLE
All i get is blank pages

### ISSUE THREAD

**Issue by jamieduk** [ron-maxweb/pwned-passwords#14]
Im just getting blank pages, things should just work out of the box but with php password checks for have i been pwned just shows blank pages and warnings upon install saying its outdated! i expect a page with a form a user can put in there password and then it will be checked i  need this done in plain text as i dont use sha or md5 i use a custom password algo that wont be found anywhere so plaintext or sha1 version of password may still work but at present all i get is blank pages this is not good, along with using bs libraries is also not ideal, stick to native php code no libs required and then things wont go wrong i hate using composer and other frameworks again just stick to native php code get this working all from 1 php file as your only sending data to api and getting response after all so i expect this to be fixed and all redundant bs code an libs to be removed and 1 single native php to just return is password is compromised or not when can i expect this new update?

ps iv tried 2 now an running out of patients all i want is this simple concept working for users can be secure thats all! thanks!

put in how i can do this manually if you wont update your old crappy code please. i dont mind doing it just explain how i could do it how i describe thanks

**Comment by jamieduk** [comment]
So make it work with something like this style ok

```
<?php
ini_set("display_errors", "1");
error_reporting(E_ALL);

$output="";

if (isset($_POST['userinput'])) {

$data=$_POST["userinput"];

$hashed=strtoupper(sha1($data));
$chars=substr($hashed, 0, 5);

$lines_array=file("https://api.pwnedpasswords.com/range/$chars");

$hits=0;
$find=false;

foreach ( $lines_array as $line) {

$result=explode(':', $line);
$hash=$chars . $result[0];
$hits=intval($result[1]);

if ($hash === $hashed) {
$find=true;
break;
}
}

if(!$find){
$hits=0;
}

// determin hits
if ($hits > 0) {
$output="<center>This Password Has Been Found $hits Times,<br><b>So Do NOT Use This Password!!!</b>";
} else {
$output="<center> Password Seems To Be Safe!<br><b>(For Now At Least) Check Regularly!</b>";
}

}

?>
<div style="clear:both;height:24px;"></div>
<center>
<div style="float:center;height:auto;width:500px;background:#3366FF;margin-left:25px;margin-right:5px;border:#CCC 2px solid;">
<p><br>

<h2>Check Your Password</h2>

<?php echo $output; ?>

<center>
<form name="form" method="post">
<input type="password" name="userinput" size="50" required="" value=""/>
<input type="submit" id="search-submit" value="Submit" style="cursor:pointer;"/>
</form>

</center>

</div>

</center>


```


no libs or frames needed just make it happen! thats how to code!

**Comment by mxrxdxn** [comment]
I don't even know where to start with this.

Just because you don't understand the concept of the library does not make it "crappy code". You should question your own abilities if you cannot understand how Composer and third party packages function, and if you cannot follow a simple readme which tells you in layman's terms how to use said library.

I also take it as an insult that you dare come here complaining about my code when you then post the above? That's some of the worst fucking code I've ever read. I sincerely hope you are not employed in any way, shape or form in the web development world, because you are a walking unmitigated disaster waiting to happen.

Have a great day.
